jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -117,6 +118,8 @@ public void executeTimerJob(Timer timer) {
         }
         try {
             transaction(this::executeTimerJobInstance, timerJobInstance);
+        } catch(SessionNotFoundException e) {
```

Comment:
This is a good idea, unfortunately the SessionNotFoundException is wrapped in a RuntimeException:

22:58:12,758 ewih ERROR [stderr] (EJB default - 5) java.lang.RuntimeException: org.kie.internal.runtime.manager.SessionNotFoundException: No session found for context 8315094
22:58:12,758 ewih ERROR [stderr] (EJB default - 5)         at deployment.ROOT.war//org.jbpm.process.core.timer.impl.GlobalTimerService.getRunner(GlobalTimerService.java:293)

I'd suggest to remove the exception wrapping here:
https://github.com/kiegroup/jbpm/blob/master/jbpm-flow/src/main/java/org/jbpm/process/core/timer/impl/GlobalTimerService.java#L293

Reply:
rather compute the exception is session not found... it could go through very deep in the stacktrace